master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calicoctl.yaml

```diff
@@ -13,10 +13,76 @@ metadata:
   namespace: kube-system
 spec:
   hostNetwork: true
+  serviceAccountName: calicoctl
   containers:
   - name: calicoctl
     image: quay.io/calico/ctl:{{site.data.versions[page.version].first.components["calicoctl"].version}}
     command: ["/bin/sh", "-c", "while true; do sleep 3600; done"]
     env:
     - name: DATASTORE_TYPE
       value: kubernetes
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: calicoctl
+rules:
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - pods
+      - nodes
+    verbs:
+      - get
+      - list
+      - update
```

Comment:
Do we actually need `update` on pods? I thought we just needed it on `pods/status`

Reply:
Do we not need it to create/modify labels on k8s pods (I thought we did that I may be wrong :) )

Reply:
nope, no modifications to the spec or metadata.